packages/astro/src/content/vite-plugin-content-imports.ts

```diff
@@ -371,6 +371,7 @@ function stringifyEntryData(data: Record<string, any>): string {
 			});
 		} else {
 			throw new AstroError({
+				name: 'ContentImportsPluginError',
 				message: 'Unexpected error processing content collection data.',
 			});
```

Comment:
I'll note that this should be a defined error in error-data with perhaps common causes and stuff. But something to see in another PR perhaps.

The only time where throwing an AstroError that is not defined is ok is if the error is temporary (ex: experimental feature has not been enabled).